org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/nexus/SolsticeScanMonitor.java

```diff
@@ -164,21 +196,48 @@ public NXcollection createNexusObject(NexusScanInfo info) {
 		
 		return scanPointsCollection;
 	}
-
+	
+	private String durationInMillisToString(Duration duration) {
+		long days = duration.toDays();
+		duration = duration.minusDays(days);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "duration". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1226)